azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/compute/functions/DeploymentToNodeMetadata.java

```diff
@@ -144,22 +132,9 @@ public NodeMetadata apply(final VMDeployment from) {
       }
 
       Credentials credentials = credentialStore.get("node#" + from.deployment.name());
-      if (credentials != null && credentials.identity.equals(JCLOUDS_DEFAULT_USERNAME)) {
-          credentials = new Credentials(AZURE_LOGIN_USERNAME, credentials.credential);
-      }
-      else if (credentials == null) {
-         String username = AZURE_LOGIN_USERNAME;
-         String password = AZURE_LOGIN_PASSWORD;
-         if (username == null) {
-            username = "jclouds";
-         }
-         if (password == null) {
-            password = "Password1!";
-         }
-
-         credentials = new Credentials(username, password);
+      if (credentials != null) {
+         builder.credentials(LoginCredentials.fromCredentials(credentials));
       }
```

Comment:
Images created by jclouds will always have the credentials now. Either the image default credentials (those configured in the properties) or the ones configured by the user.
If there are no credentials in the credential store, we shouldn't set the default ones, as we really don't know if those are the good credentials (it could be a node that has been manually deployed in the Azure portal). When uncertain about the credentials, we'd better don't set them and let the user manually provide them when running scripts.